index.html

```diff
@@ -1,11 +1,94 @@
 <!DOCTYPE html>
 <html lang="en">
   <head>
-    <meta charset="UTF-8">
-    <title>Fork Me! FCC: Test Suite Template</title>
-    <link rel="stylesheet" href="css/style.css">
+    <meta charset="UTF-8" />
+    <title>Tribute page</title>
+    <link rel="stylesheet" href="css/style.css" />
   </head>
+  <p id="title">Dr. Norman Borlaug</p>
+  <p id="title-info">The man who saved a billion lives</p>
```

Comment:
It is a good practice to use a semantic "header" element as the container and use "h1" element instead  of p on line #8.